Tribler/util.py

```diff
@@ -0,0 +1,41 @@
+"""
+This file contains various utility methods.
+"""
+import sys
+from collections import namedtuple
+
```

Comment:
`StringIO`, `is_long_or_int` and part of `urllib_future` are already defined in:
https://github.com/Tribler/py-ipv8/blob/master/ipv8/util.py

Please reuse/extend those to get rid of code duplication.

Reply:
Do we actually want to depend on the IPv8 submodule for these variables? Although it has some code duplication, I think it makes sense to also have them in the Tribler code base.

Reply:
I am struggling with why we use __six__ and then rewrite much of the same functionality in custom code in __util.py__.  Why not leverage __six__ as much as possible?

Reply:
__six.moves__ does so much of this in a more common way.

Reply:
@qstokkink could you comment on the usage of `six`? In IPv8, you only used it for metaclasses right?

Reply:
This is related to https://github.com/Tribler/tribler/pull/4096#issuecomment-445440146.

**tl;dr** I agree, we should use `six` as much as possible. As I missed some of `six`'s functionality when porting IPv8, I also sinned by making duplicate functionality (see https://github.com/Tribler/py-ipv8/issues/382).

Reply:
Ok, I will change my PR to use `six` instead.